manpages/mkfs.fat.8.in

```diff
@@ -20,176 +20,174 @@
+If nothing is specified, \fBmkfs.fat\fP will automatically select between 12, 16
 and 32 bit, whatever fits better for the filesystem size.
-.IP "\fB\-h\fR \fINUMBER-OF-HIDDEN-SECTORS\fR" 4
+.IP "\fB\-h\fP \fINUMBER-OF-HIDDEN-SECTORS\fP" 4
 Select the number of hidden sectors in the volume.
 Apparently some digital cameras get indigestion if you feed them a CF card
 without such hidden sectors, this option allows you to satisfy them.
-.IP "\fB\-i\fR \fIVOLUME-ID\fR" 4
-Sets the volume ID of the newly created filesystem; \fIVOLUME-ID\fR is a 32-bit
+.IP "\fB\-i\fP \fIVOLUME-ID\fP" 4
+Sets the volume ID of the newly created filesystem; \fIVOLUME-ID\fP is a 32-bit
 hexadecimal number (for example, 2e24ec82).
 The default is a number which depends on the filesystem creation time.
-.IP "\fB\-I\fR" 4
+.IP "\fB\-I\fP" 4
 It is typical for fixed disk devices to be partitioned so, by default, you are
 not permitted to create a filesystem across the entire device.
-\fBmkfs.fat\fR will complain and tell you that it refuses to work.
+\fBmkfs.fat\fP will complain and tell you that it refuses to work.
 This is different when using MO disks.
 One doesn't always need partitions on MO disks.
 The filesystem can go directly to the whole disk.
-Under other OSes this is known as the 'superfloppy' format.
-This switch will force \fBmkfs.fat\fR to work properly.
-.IP "\fB\-l\fR \fIFILENAME\fR" 4
-Read the bad blocks list from \fIFILENAME\fR.
-.IP "\fB\-m\fR \fIMESSAGE-FILE\fR" 4
+Under other OSes this is known as the \fIsuperfloppy\fP format.
+This switch will force \fBmkfs.fat\fP to work properly.
+.IP "\fB\-l\fP \fIFILENAME\fP" 4
+Read the bad blocks list from \fIFILENAME\fP.
+.IP "\fB\-m\fP \fIMESSAGE-FILE\fP" 4
 Sets the message the user receives on attempts to boot this filesystem without
 having properly installed an operating system.
 The message file must not exceed 418 bytes once line feeds have been converted
 to carriage return-line feed combinations, and tabs have been expanded.
 If the filename is a hyphen (-), the text is taken from standard input.
-.IP "\fB\-M\fR \fIFAT-MEDIA-TYPE\fR" 4
+.IP "\fB\-M\fP \fIFAT-MEDIA-TYPE\fP" 4
 Specify the media type to be stored in the FAT boot sector.
 This value is usually 0xF8 for hard disks and is 0xF0 or a value from 0xF9 to
 0xFF for floppies or partitions to be used for floppy emulation.
-.IP "\fB\-n\fR \fIVOLUME-NAME\fR" 4
+.IP "\fB\-n\fP \fIVOLUME-NAME\fP" 4
 Sets the volume name (label) of the filesystem.
 The volume name can be up to 11 characters long.
 Supplying an empty string, a string consisting only of white space or the
-string "NO NAME" as \fIVOLUME-NAME\fR has the same effect as not giving the
-\fB\-n\fR option.
+string "NO NAME" as \fIVOLUME-NAME\fP has the same effect as not giving the
+\fB\-n\fP option.
 The default is no label.
-.IP "\fB\-\-codepage\fR=\fIPAGE\fR" 4
-Use DOS codepage \fIPAGE\fR to encode label.
+.IP "\fB\-\-codepage\fP=\fIPAGE\fP" 4
+Use DOS codepage \fIPAGE\fP to encode label.
```

Comment:
This is tricky because you need three fonts on the same input line.  In that case, there is no good macro and using font escapes is the best you can do, i.e.

    .TP
    \fB\-\-codepage\fP=\fIPAGE\fP

Yes, the man(7) language is quirky and neither beautiful nor particularly powerful nor easy to use cleanly.